package.json

```diff
@@ -4,8 +4,10 @@
   "description": "Bucklescript bindings for Puppeteer (headless chrome browser) written in Reason",
   "main": "src/Puppeteer.re",
   "scripts": {
+    "precommit": "lint-staged",
```

Comment:
We might as well run the test suite here too.

Reply:
Fixed is commit 98c222f
Added in `.lintstagedrc`

Reply:
Ah even better.